HW1/P2/P2b.py

```diff
@@ -1,3 +1,32 @@
 from P2 import *
+# from P2a import make_mandelbrot_image_and_hist
 
-# Your code here
+import findspark
+findspark.init()
+import numpy as np
+import pyspark
+
+sc = pyspark.SparkContext(appName='mandelbrot')
+
+
+x_values = sc.parallelize(np.random.permutation(2000), 10)
+y_values = sc.parallelize(np.random.permutation(2000), 10)
+coords = x_values.cartesian(y_values)
+
+def make_mandelbrot_image_and_hist(coords_rdd, hist_fname):
```

Comment:
There's no good reason, the import wasn't working for some reason and I ended up copying and pasting rather than actually fixing it, definitely bad practice.